tasks.py

```diff
@@ -20,6 +22,9 @@
 STACK_NAME = env('STACK_NAME')
 PROD_IDENTIFIER = "prod"
 NONINTERACTIVE = env('NONINTERACTIVE')
+ZOOM_API_KEY = env('ZOOM_API_KEY')
```

Comment:
Along the way I've tried to stick to using `getenv(...)` within the task code vs defining these at the top. The advantage being that `getenv` complains properly if a necessary value isn't set (or a default given). I forgot to set my zoom base api and got a non-helpful stacktrace.